### PR TITLE
Update time_complexity.md in TS and JS

### PR DESCRIPTION
**Pull request by leeguooooo**
增加对应的 js 和 ts 代码，方便前端学习的同学阅读代码。

> Tip: If this PR is not related to the coding or code translation, please ignore the checklist.

### Checklist

- [x] I've tested the code and ensured the outputs are the same as the outputs of reference codes.
- [x] I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
- [x] The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

### CODE REVIEW

File: docs/chapter_computational_complexity/time_complexity.md [hunk 1]
```diff
@@ -79,13 +79,31 @@ $$
...
+    }
     ```
 
 === "TypeScript"
 
     ```typescript title=""
-
+    // 在某运行平台下
+    function algorithm(n: number): void {
+      let a = 2 // 1 ns
```
Comment (krahets) [inline docs/chapter_computational_complexity/time_complexity.md:99]: Four spaces is required for the indentation in this project.

**Review by krahets — COMMENTED** [review]
Thank @leeguooooo ! Could you please create source code files `time_complexity.js` , `time_complexity.ts` and test the source code? You can refer to other JS and TS codes to see more information.

File: docs/chapter_computational_complexity/time_complexity.md (hunk 1 above)
Comment (leeguooooo) [reply docs/chapter_computational_complexity/time_complexity.md]: I have fixed it.

**Comment by leeguooooo** [comment]
> Thank @leeguooooo ! Could you please create source code files `time_complexity.js` , `time_complexity.ts` and test the source code? You can refer to other JS and TS codes to see more information.

The relevant code has been provided and has been tested and passed.

File: codes/javascript/chapter_computational_complexity/time_complexity.js [hunk 2]
```diff
@@ -0,0 +1,157 @@
+// File: time_complexity.js
+// Created Time: 2022-12-21
+// Author: leeguooooo (leeguooooo@gmail.com)
+
+/* 常数阶 */
+function constant(n) {
+    let count = 0
```
Comment (krahets) [inline codes/javascript/chapter_computational_complexity/time_complexity.js:7]: Lack of `;` in the end of every line.

**Comment by justin-tse** [comment]
> 增加对应的 js 和 ts 代码，方便前端学习的同学阅读代码。
> 
> > Tip: If this PR is not related to the coding or code translation, please ignore the checklist.
> 
> ### Checklist
> * [x]  I've tested the code and ensured the outputs are the same as the outputs of reference codes.
> * [x]  I've checked the codes (formatting, comments, indentation, file header, etc) carefully.
> * [x]  The code does not rely on a particular environment or IDE and can be executed on a standard system (Win, macOS, Ubuntu).

Thank you @leeguooooo, please make Checklist is right. I found the following is not correct.
 Please test the code and ensured the outputs are the same as the outputs of reference codes. (You should follow the Java code, which is the standard for our translation)
Please check the codes (formatting, comments, indentation, file header, etc) carefully.

**Comment by krahets** [comment]
The PR is closed due to no response from the author.

**Pull request closed without merging**